File: src/ApiServer/AzW.Model/Provision/VNet.cs

using System.Collections.Generic;

namespace AzW.Model
 {
     public class VNet : Resource
     {
          public string Location { get; set; }
          public string ResourceGroupName { get; set; }
          public string AddressSpace { get; set; }
         public List<Subnet> Subnets { get; set; }
     }
 }

File: src/Portal/src/models/VNet.js
import ResourceType from './ResourceType'
export default class VNet
{
    constructor (){
        this.GraphModel = {
            Id: '',
            ResourceType: ResourceType.VNet(),
            DisplayName: '',
            SubnetsAndCidrs: []
        };

        this.ProvisionContext = {
            ResourceType: ResourceType.VNet(),

            Name: 'vnet-internet',
            Location: 'westus',
            ResourceGroupName: '',
            AddressSpace: '10.0.0.0/16',
            Subnets: []
        };
        this.CalculatorContext = {
            IsFree: true,
            Tier: ''
        }
    }
}